routing/index_graph.hpp

```diff
@@ -76,13 +207,171 @@ class IndexGraph final
     m_jointIndex.ForEachPoint(jointId, forward<F>(f));
   }
 
+  /// \brief Add restrictions in |restrictions| to |m_ftPointIndex|.
+  void ApplyRestrictions(RestrictionVec const & restrictions);
+
+  /// \returns RoadGeometry by a real or fake featureId.
+  RoadGeometry const & GetRoad(uint32_t featureId);
+
+  bool IsFakeFeature(uint32_t featureId) const { return featureId >= kStartFakeFeatureIds; }
```

Comment:
Yes. It returns true for fake feature (of IndexGraph) and for additional feature (of IndexGraphStarter) as well.